grapher/Models/Serialized/GUISettings.cs

```diff
@@ -10,32 +10,35 @@ public class GUISettings
 
         public GUISettings() {}
 
-        public GUISettings(bool autoWrite, int dpi, int pollRate)
-        {
-            AutoWriteToDriverOnStartup = autoWrite;
-            DPI = dpi;
-            PollRate = pollRate;
-        }
-
         #endregion Constructors
 
         #region Properties
 
-        [JsonProperty(Order = 1)]
-        public bool AutoWriteToDriverOnStartup { get; set; }
 
-        [JsonProperty(Order = 2)]
+        [JsonProperty(Order = 1)]
         public int DPI { get; set; }
 
-        [JsonProperty(Order = 3)]
+        [JsonProperty(Order = 2)]
         public int PollRate { get; set; }
 
-        [JsonProperty(Order = 4)]
+        [JsonProperty(Order = 3)]
         public bool ShowLastMouseMove { get; set; }
 
         [JsonProperty(Order = 4)]
         public bool ShowVelocityAndGain { get; set; }
 
         #endregion Properties
+
+        #region Methods
+
+        public bool ValueEquals(GUISettings other)
```

Comment:
Is this a sneaky way to avoid overriding Equals and GetHashCode? You should just overwrite them even if it seems unneeded.